Spec/Test Utilities/NSObject+TestSuite.m

```diff
@@ -17,7 +17,7 @@ @implementation NSObject (TestSuite)
     return [self aspect_hookSelector:selector withOptions:AspectPositionAfter usingBlock:^(id<AspectInfo> info) {
         __autoreleasing id arg;
         [[info originalInvocation] getArgument:&arg atIndex:2+index];
-        callback([arg copy]);
+        callback(arg);
```

Comment:
Why would they use copy here? 🤔 Just curious

Reply:
I'm guessing that @lawrence-forooghian has removed this because `copy` is generally shallow in nature. 🤔 ... Regardless, this is test suite code so not a product implementation change. 😄

Reply:
I'm going to resolve this conversation on @maratal's behalf, so that I can land this PR. Please feel free to continue the conversation or create an issue if you feel there is more work to be done on the test suite later. Thanks.

Reply:
I'd actually addressed this in my [commit message](https://github.com/ably/ably-cocoa/pull/1266/commits/dd272cfb76d5337c178a6e4a15250603db4819bb).